data/recommendation/style.scss

```diff
@@ -22,7 +22,6 @@ body {
     color: black;
     }
   & > div {
-    height: 40%;
     align-content: space-between;
     align-items: center;
     display: flex;
```

Comment:
I use `display: flex` on all three sibling divs, where one contains the image, one contains the text, and one contains the install button, so that they can all align vertically.